lib/WWW/Mechanize.pm

```diff
@@ -510,6 +513,29 @@ sub _SUPER_put {
     return $self->request( HTTP::Request::Common::PUT( @parameters ), @suff );
 }
 
+=head2 $mech->head ($uri )
+
+Performs a HEAD request to I<$uri>. Returns an L<HTTP::Response> object.
+I<$uri> can be a well-formed URI string, a L<URI> object, or a
+L<WWW::Mechanize::Link> object.
+
+=cut
+
+sub head {
+   my $self = shift;
```

Comment:
Looks like the spacing is off by one here? Maybe we could just run the new code through `perltidy`?